components/ws-manager-bridge/src/bridge.ts

```diff
@@ -94,13 +96,43 @@ export class WorkspaceManagerBridge implements Disposable {
             this.config.controllerMaxDisconnectSeconds,
         );
 
+        const tim = setInterval(() => {
+            this.updateWorkspaceClasses(cluster, clientProvider);
+        }, controllerIntervalSeconds * 1000);
+        this.disposables.push({ dispose: () => clearInterval(tim) });
+
         log.info(`Started bridge to cluster.`, logPayload);
     }
 
     public stop() {
         this.dispose();
     }
 
+    protected async updateWorkspaceClasses(clusterInfo: WorkspaceClusterInfo, clientProvider: ClientProvider) {
```

Comment:
nit / fyi. we no longer use `protected` but `private` if a method is only used locally. We are opportunistically changing that as we touch classes.

Reply:
Where can I find the list of opportunistic improvements? 
So far there are two:
- `JSON` over `TEXT`
- `private` over `protected`

Reply:
We communicate them on slack and then educate each other in code reviews